File: problemas/iniciante/2936/2936.exs
curupira = IO.gets("") |> String.trim |> String.to_integer
boitata = IO.gets("") |> String.trim |> String.to_integer
boto = IO.gets("") |> String.trim |> String.to_integer
mapinguari = IO.gets("") |> String.trim |> String.to_integer
iara = IO.gets("") |> String.trim |> String.to_integer

IO.puts (225 + 300 * curupira + 1500 * boitata + 600 * boto + 1000 * mapinguari + 150 * iara)

File: problemas/iniciante/2936/2936.clj
(let [curupira (read-string (read-line))
      boitata (read-string (read-line))
      boto (read-string (read-line))
      mapinguari (read-string (read-line))
      iara (read-string (read-line))]
  (printf "%d\n" (+ 225 (* 300 curupira) (* 1500 boitata) (* 600 boto) (* 1000 mapinguari) (* 150 iara))))